.github/workflows/dev_cron.yml

```diff
@@ -36,6 +39,7 @@ jobs:
             const path = ".github/workflows/pull_request/jira_link.js";
```

Comment:
Could you also move `.github/workflows/pull_request/*` to `.github/wokflows/dev_cron/`?

Reply:
Done. BTW we should act on event instead of using cron tasks https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-event-pull_request

Reply:
We can't run this on `pull_request` event.
Because we don't have writable `GITHUB_TOKEN` by pull request from other repository.

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-events-for-forked-repositories

> The permissions for the `GITHUB_TOKEN` in forked repositories is read-only.

If we use `on.schedule.cron`, we can use writable `GITHUB_TOKEN`.

Reply:
I'm interpreting that as that the triggered workflow runs on the base repository, thus it uses the (writable) GitHub token. The read-only note is about you cannot do nasty GitHub API calls acting from forks. 
I assume you've tried the full request event trigger though.

Reply:
Yes. I tried `pull_request` action first. :-)

This workflow may comment to pull requests on the base repository.
If this wokflow is ran on `pull_request`, this workflow uses configuration in the pull request not in the base repository. If we have a writable `GITHUB_TOKEN` for `pull_request` event, evil fork repository can write any comments by putting evil code to `.github/workflows/dev_cron/*.js`.